### PR TITLE
Updated package_examples.rb to reflect the changes to Swift packaging

### PR DESCRIPTION
**Pull request by jpsim**
/cc @segiddins

### CODE REVIEW

**Comment by jpsim** [comment]
I aborted the CI build because it never calls `package_examples.rb`.

File: scripts/package_examples.rb [hunk 1]
```diff
@@ -36,26 +36,41 @@ def remove_all_dependencies(project_path)
...
+    file.puts contents.gsub("/build/ios", "/ios/static")
+                      .gsub("/build/osx", "/osx")
+  end
+end
+
+# Remove RealmSwift target and dependencies from all example swift projects
+
+swift_examples = [
+  "examples/ios/swift/RealmExamples.xcodeproj",
+  "examples/osx/objc/RealmExamples.xcodeproj"
```
Comment (segiddins) [inline scripts/package_examples.rb]: this is `objc`

**Comment by jpsim** [comment]
This isn't the proper fix, so I'm marking this PR as WIP until I can find an appropriate solution.

**Comment by jpsim** [comment]
I've found the proper fix now, @segiddins please review again. I'll let the CI PR job run this time because I had to make changes to the Swift examples Xcode project.

File: examples/ios/swift/RealmExamples.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -184,7 +184,7 @@
 /* Begin PBXFileReference section */
 		0295B91519D103A70036D6C3 /* libc++.dylib */ = {isa = PBXFileReference; lastKnownFileType = "compiled.mach-o.dylib"; name = "libc++.dylib"; path = "usr/lib/libc++.dylib"; sourceTree = SDKROOT; };
 		3FC898FE1A1414110067CBEC /* ViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; lineEnding = 0; path = ViewController.swift; sourceTree = "<group>"; xcLanguageSpecificationIdentifier = xcode.lang.swift; };
-		C07E5D791B15001500ED625C /* Realm.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Realm.framework; path = ../../build/ios/swift/Realm.framework; sourceTree = "<group>"; };
```
Comment (segiddins) [inline examples/ios/swift/RealmExamples.xcodeproj/project.pbxproj]: This seems wrong to me

Comment (jpsim) [inline examples/ios/swift/RealmExamples.xcodeproj/project.pbxproj]: I had missed that we're now copying `build/ios-dynamic/Realm.framework` into `build/ios/swift/`. I'll update this.

**Comment by segiddins** [comment]
:+1:

**Pull request merged**